dolfyn/rotate/base.py

```diff
@@ -77,11 +86,12 @@ def euler2orient(heading, pitch, roll, units='degrees'):
        - a "ZYX" rotation order. That is, these variables are computed
          assuming that rotation from the earth -> instrument frame happens
          by rotating around the z-axis first (heading), then rotating
-         around the y-axis (pitch), then rotating around the x-axis (roll).
+         around the y-axis (pitch), then rotating around the x-axis (roll). 
+         Note this requires matrix multiplication in the reverse order.
```

Comment:
Good note! Why don't you also add this as a comment above `return np.einsum(...)` at the end of this function. Something like:

    # As mentioned in the docs, the matrix-multiplication order is "reversed" (i.e., ZYX
    # order of rotations happens by multiplying R*P*H).
    # It helps to think of this as left-multiplying omat onto a vector. In which case,
    # H gets multiplied first, then P, then R (i.e., the ZYX rotation order).

Reply:
Done.